packages/graphiql-react/vite.config.ts

```diff
@@ -26,11 +27,18 @@ export default defineConfig({
     sourcemap: true,
     lib: {
       entry: 'src/index.ts',
-      fileName: 'graphiql-react',
+      fileName: 'index',
       formats: ['cjs', 'es'],
     },
     rollupOptions: {
-      external: ['graphql', 'react', 'react-dom', 'react/jsx-runtime'],
+      external: [
+        'react/jsx-runtime',
+        // Exclude peer dependencies and dependencies from bundle
+        ...Object.keys(packageJSON.peerDependencies),
+        ...Object.keys(packageJSON.dependencies).filter(
+          dependency => dependency !== 'codemirror',
```

Comment:
is there a reason we leave codemirror?

Reply:
yeah there were some errors from `graphiql` without this filter, anyway we'll remove it with Monaco migration

Reply:
I think when we replace webpack with vite in graphiql we can get rid of this beforehand